lambda/controller/create-replica/index.js

```diff
@@ -24,26 +24,42 @@ exports.handler = function(event, context, callback){
       return callback(err);
     }
 
+    var billingMode = 'PROVISIONED';
+
+    // Check the billing mode of the table
+    if (data.Table.hasOwnProperty('BillingModeSummary')){
+      if (data.Table.BillingModeSummary.BillingMode == 'PAY_PER_REQUEST') {
```

Comment:
I'd recommend using === instead of == to avoid loose matching results. It won't make a material difference here, but it's good practice and standard in all our modern JS projects.